recipeApp/views.py

```diff
@@ -1,3 +1,66 @@
 from django.shortcuts import render
+from django.contrib.auth.models import User
+from django.views.generic import TemplateView, ListView, DetailView
+from .models import Recipe, RecipeCollection
+from django.core.paginator import Paginator
 
-# Create your views here.
+
+class HomePageView(TemplateView):
+    template_name = 'home.html'
+    
+
+class RecipeListView(ListView):
+    model = Recipe
+    template_name = 'recipes/list.html'
+    context_object_name = 'recipes'
+    paginate_by = 3
+
+    def get_queryset(self):
+        return Recipe.objects.filter(featured=False).order_by('-created_at')
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        
+        featured_recipes = Recipe.objects.filter(featured=True).order_by('-created_at')
```

Comment:
Don't you have to filter recipes by user also ?

Reply:
Can I add in seperate page or in same page I can add my recipe seperately sir ?
I thought I can filter the recipe by user in filter functionality sir